package.json

```diff
@@ -73,7 +73,9 @@
     "test:coverage": "nyc --reporter=html --reporter=text mocha -R spec ./test/functional ./test/unit",
     "test-travis": "eslint . && npm run test:coverage",
     "test-only": "mocha ./test/functional ./test/unit",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "build-docker": "docker build -t verdaccio .",
```

Comment:
Yes, it expects the docker to be installed on the machine. This is also required if you want to follow the current instructions.